app/models/miq_group.rb

```diff
@@ -342,6 +342,10 @@ def current_user_group?
     id == current_user_group.try(:id)
   end
 
+  def destroy_subscribed_widget_sets
+    MiqWidgetSet.where(:group_id => id).destroy_all
```

Comment:
So if we delete this group, a user widget set will probably want to stick around.
So if `user_id` is around, want to figure out the user's new group and set the widget set's group to the user's new set?

--
I do want a test around user_id and group_id set

Reply:
update: if we destroy this group, destroy any widget set linking to this